<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>260</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Select game:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="smallGameButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Small</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="mediumGameButton">
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="largeGameButton">
      <property name="text">
       <string>Large</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="CustomGameButton">
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Width(5 to 50):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="WidthEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Height(5 to 50):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="HeightEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Number of mines(4 to 2250):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NumberOfMinesEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="SaveSettingsButton">
      <property name="text">
       <string>Save settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CustomGameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidthEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomGameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomGameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>HeightEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomGameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomGameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>NumberOfMinesEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CustomGameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
